rivanna/README.md

```diff
@@ -0,0 +1,183 @@
+# Running Cylon on Rivanna
+
+Arup Sarker (arupcsedu@gmail.com, djy8hg@virginia.edu)

```

Comment:
Generally don't put author names unless it is a blog post or an article. But I am not very much informed with the recent practices, so I could be wrong. :slightly_smiling_face:

Reply:
I followed this convention by taking the sample from Summit compatibility documents. 
https://github.com/cylondata/cylon/tree/main/summit#readme